modules/openstack/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/go-logr/logr v1.2.4
-	github.com/gophercloud/gophercloud v1.2.0
+	github.com/gophercloud/gophercloud v1.4.0
```

Comment:
@stuggi FYI bumping gophercloud to 1.4.0 to get [1] in.

[1] https://github.com/gophercloud/gophercloud/pull/2616

Reply:
I would say it's fine if everything still works :)